CircleandCross2/Program.cs

```diff
@@ -21,65 +21,75 @@ static void ShowGameBoard (string[,] board)
 			
 		}
 		
-		static bool CheckWinCondition(string[,] board)
-		{
+		
+		static bool CheckWinCondition(string[,] board, string PlayerToken)
+					{
 			for (int columns = 0; columns < 3; columns++)
 			{
-				if (board[columns, 0] == "[o]" && board[columns, 1] == "[o]" && board[columns, 2] == "[o]"
-					|| board[columns, 0] == "[x]" && board[columns, 1] == "[x]" && board[columns, 2] == "[x]")
-					
+				if (board[columns, 0] == PlayerToken && board[columns, 1] == PlayerToken && board[columns, 2] == PlayerToken)
+										
 				{
 					return true;
 				}
-
 			}
 
 			for (int rows = 0; rows < 3; rows++)
 			{
-				if (board[0, rows] == "[o]" && board[1, rows] == "[o]" && board[2, rows] == "[o]"
-					|| board[0, rows] == "[x]" && board[1, rows] == "[x]" && board[2, rows] == "[x]")
+				if (board[0, rows] == PlayerToken && board[1, rows] == PlayerToken && board[2, rows] == PlayerToken)
 
 				{
 					return true;
 				}
 
 			}
-			if (board[0, 0] == "[o]" && board[1, 1] == "[o]" && board[2, 2] == "[o]"
-				|| board[0, 0] == "[x]" && board[1, 1] == "[x]" && board[2, 2] == "[x]"
-				|| board[0, 2] == "[o]" && board[1, 1] == "[o]" && board[2, 0] == "[o]"
-				|| board[0, 2] == "[x]" && board[1, 1] == "[x]" && board[2, 0] == "[x]")
+			if (board[0, 0] == PlayerToken && board[1, 1] == PlayerToken && board[2, 2] == PlayerToken				
+				|| board[0, 2] == PlayerToken && board[1, 1] == PlayerToken && board[2, 0] == PlayerToken)
 			{
 				return true;
 			}
 			return false;
 		}
 
-		//static bool IsCorrectInput(string text,string text2, string[,] board)
-		//{
-		//	int position1i;
-		//	int position1j;
-		//	bool isNumber = int.TryParse(text, out position1i);
-		//	bool isNumber2 = int.TryParse(text2, out position1j);
+		static Position ReturnPosition(string [,]  array)
```

Comment:
This function can also have an additional parameter for player token. Then it can be easily used for players 'x' and 'o'.

Reply:
Please write whether I am going in the right direction.I had problem becouse i dont founded nothing similar ( class which return 2 values).I m not sure that i have to use in class: Console Readline and int.Parse

Reply:
Please write whether I am going in the right direction..I had problem becouse i dont founded nothing similar ( class which return values in method))

Reply:
Now Position class looks better. You just need to return an object of that type in the ReturnPostion method

Reply:
i did it but i am not sure whether it is good becouse when i use method returnposition in main in line 134, then i have not "o" in board.

Reply:
You don't use the return value from that method.
```csharp
var postion = ReturnPosition(array);
array[/*values from postion */] = /*something*/;
```

Reply:
i sent you my last try, but program dont work like i want. I tried a lot of solution but all was wrong